applications/GeoMechanicsApplication/custom_processes/geo_extrapolate_integration_point_values_to_nodes_process.h

```diff
@@ -67,7 +67,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoExtrapolateIntegrationPointValues
     std::vector<const Variable<Vector>*>              mVectorVariables;
     std::vector<const Variable<Matrix>*>              mMatrixVariables;
     const Variable<double>&                           mrAverageVariable       = NODAL_AREA;
-    mutable std::map<SizeType, Matrix>                mExtrapolationMatrixMap = {};
+    std::map<SizeType, Matrix>                        mExtrapolationMatrixMap = {};
```

Comment:
Removed the mutable. I know it is usual for caches, but when using it in parallel it is confusing that const functions cannot be run in parallel.